.travis.yml

```diff
@@ -13,6 +13,10 @@ matrix:
           python: 2.7
 before_install:
   - ./.travis/before_install.sh
+before_script: # configure a headless display to test plot generation
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
```

Comment:
I tried to use the Agg backedn here
https://github.com/rflamary/POT/blob/pytest/test/test_plot.py

but travis still failed with DISPLAY error
https://travis-ci.org/rflamary/POT/builds/256924206

maybe the test_plot.py is not good but I don't see the problem